CHANGES.md

```diff
@@ -1,3 +1,16 @@
+## edge-19.4.3
+
+* CLI
+  * **Fixed** `linkerd upgrade` command not upgrading proxy containers (thanks
+    @jon-walton for the issue report!)
+  * **Fixed** `linkerd upgrade` command not installing the identity service when
+    it was not already installed
+* Internal
```

Comment:
Aside from the uuid-generation, which fixes snyk reports, i don't think any of these really have a user-facing impact, so maybe just summarize as a _Thanks_ blurb at the bottom (as in 19.3.3).

And it would be fine to include the UUId in the Cli section, since it was actually cli-related